### ISSUE TITLE
Health Check Failing with Permission Denied

### ISSUE THREAD

**Issue by pierow2k** [meeb/tubesync#41]
Running the Docker image (pulled as `ghcr.io/meeb/tubesync:v0.8`) the container's Health Check fails with:
```bash
/bin/sh: 1: /app/healthcheck.py: Permission denied

```
Upon further investigation, the `/app/healthcheck.py` script does not appear to have execute permission:
```bash
# ls -la /app/healthcheck.py
-rw-r--r-- 1 root root 705 Jan 20 01:14 /app/healthcheck.py
#
```
Setting the permission manually inside the running container (with `chmod +x /app/healthcheck.py`) seems to resolve the problem.

**Comment by meeb** [comment]
Odd, the permissions are set for `/app/healthcheck.py` explicitly via the init script in https://github.com/meeb/tubesync/blob/main/config/root/etc/cont-init.d/10-tubesync#L23

I just pulled a fresh copy of the v0.8 image and it is seemingly set correctly:

```bash
$ docker run --rm --name tubesync --env-file dev.env -ti ghcr.io/meeb/tubesync:v0.8 /bin/bash
Unable to find image 'ghcr.io/meeb/tubesync:v0.8' locally
v0.8: Pulling from meeb/tubesync
(truncated)
[services.d] done.
root@821ea006680c:/app# [2021-01-25 02:20:29 +0000] [436] [INFO] Starting gunicorn 20.0.4
[2021-01-25 02:20:29 +0000] [436] [INFO] Listening at: http://127.0.0.1:8080 (436)
[2021-01-25 02:20:29 +0000] [436] [INFO] Using worker: sync
[2021-01-25 02:20:29 +0000] [473] [INFO] Booting worker with pid: 473
^C
root@821ea006680c:/app# ls -lat /app/healthcheck.py 
-rwxr-x--x 1 root app 705 Jan 20 07:14 /app/healthcheck.py
```

Is there anything else unusual about your deployment and are you sure you're running the v0.8 image? The only way I can see this occurring is if somehow the init script didn't run which really should break a bunch of stuff (like downloading as the UID etc. would be wrong).

I'll make some minor tweaks to the init process to make it more reliable anyway in a few minutes which will be in the latest image and bundled into the upcoming v0.9 release. Even if I can't replicate your issue it should solve it for you.

**Comment by MatthK** [comment]
I just downloaded v0.8 today as well and also had the healthcheck issue. I opened a console and gave the /app/healthcheck.py the execute permission and the error disappeared.

**Comment by meeb** [comment]
This is fixed in `:latest` but not in `:v0.8` which is why you are seeing it. Try `:latest` or wait for the upcoming `v0.9` release. Note `:latest also contains other features and current dev work so will be less stable.
